lib/manageiq/appliance_console/message_configuration.rb

```diff
@@ -4,7 +4,8 @@
 module ManageIQ
   module ApplianceConsole
     class MessageConfiguration
-      attr_reader :username, :password,
+      attr_reader :username, :password, :secure,
```

Comment:
`secure` and `server_host_is_ipaddr` shouldn't be listed here since they're methods, right?

Reply:
`secure?` and `server_host_is_ipadd?` are methods but
#deactive resets `secure` and `server_host_is_ipaddr` back to nil. So I think they need to be here.

Reply:
`#deactivate` is setting `@secure = nil`, so you shouldn't need the `attr_reader`

Reply:
These attr_readers have been removed.